### PR TITLE
feature: support non-standard unix socket

### PR DESCRIPTION
**Pull request by murrant**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

**Comment by laf** [comment]
One issue with build-base which I'll push a change for.

What about *-wrapper.py scripts?

**Comment by laf** [comment]
Oh and html/install.php

**Comment by laf** [comment]
Actually build-base.php needs updating to take config options from install.

**Comment by murrant** [comment]
Need to check the python scripts.
I'm not sure about build-base.php, I thought it was ok as it is in this PR, we do not pull in dbFacile.php in that file at this time.

**Comment by laf** [comment]
I'll have to re-run it but it complains about needing an int not a string for the port.

**Comment by murrant** [comment]
OK, that's easy, I'll  check it.

File: includes/dbFacile.php [hunk 1]
```diff
@@ -17,6 +17,23 @@
  * 3. Oh, and dbFetchAll() is now dbFetchRows()
  */
 
+function dbConnect($host = null, $user = '', $password = '', $database = '', $port = null, $socket = null)
```
Comment (laf) [inline includes/dbFacile.php]: Does it not make more sense to not pass all these params and just call $config[] from within the function?

Comment (f0o) [inline includes/dbFacile.php]: the installer & co doesnt have $config present at the time of execution AFAIK

Comment (murrant) [inline includes/dbFacile.php]: Also, you aren't using this function anywhere.

**Pull request merged**